apps/webservice/src/app/[workspaceSlug]/(app)/(deploy)/(raw)/systems/[systemSlug]/(raw)/environments/[environmentId]/resources/ResourcesPageContent.tsx

```
@@ -0,0 +1,467 @@
+            ))}
+          {isLoading &&
+            Array.from({ length: 8 }).map((_, index) => (
+              <Skeleton key={index} className="h-[196px] w-full" />
+            ))}
+        </div>
+      )}
+      {selectedView === "list" && <ResourceTable resources={resources} />}
+
+      <div className="mt-4 flex items-center justify-between text-sm text-neutral-400">
+        <div>
+          {resources.length === resources.length ? (
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)